rules/windows/process_creation/proc_creation_win_whoami_priv.yml

```
@@ -6,7 +6,7 @@ references:
     - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/whoami
 author: Florian Roth (Nextron Systems)
 date: 2021/05/05
-modified: 2022/05/13
+modified: 2023/02/03
```

```suggestion
modified: 2022/05/13
```
we shouldn't change the modified date for cosmetic changes